x-pack/plugins/observability/public/pages/rules/components/status_context.tsx

```diff
@@ -18,19 +18,26 @@ import { statusMap } from '../config';
 
 export function StatusContext({
   item,
+  disabled = false,
   onStatusChanged,
   enableRule,
   disableRule,
   muteRule,
+  unMuteRule,
 }: StatusContextProps) {
   const [isPopoverOpen, setIsPopoverOpen] = useState(false);
   const [isUpdating, setIsUpdating] = useState(false);
   const togglePopover = useCallback(() => setIsPopoverOpen(!isPopoverOpen), [isPopoverOpen]);
 
-  const currentStatus = item.enabled ? RuleStatus.enabled : RuleStatus.disabled;
+  let currentStatus: RuleStatus;
+  if (item.enabled) {
+    currentStatus = item.muteAll ? RuleStatus.snoozed : RuleStatus.enabled;
```

Comment:
Is it possible to have a rule `enabled` and `muted` at the same time?

I assume `snoozed` == `muteAll`, right? The API that provides the `item` is using `muteAll`

Reply:
@fkanout Yep we are using muteAll for the snoozed (indefinitely) state. And yes a rule can be enabled and muted at the same time. I had similar questions here https://github.com/elastic/kibana/issues/123580#issuecomment-1059983258 and we ended up with @katrin-freihofner on following:
- If it is only enabled the `Enabled` state is shown in the UI
- If it is snoozed, then the `Snoozed indefinitely` state is shown in the UI, and yep it is enabled as well. 
- To unmute/unsnooze user needs to select enabled (behind the scenes it enables and unmutes the rule)

Reply:
@mgiota, excellent explanation, thanks!
`snoozed` + `enabled` will keep the rule but it will not generate alerts?